src/battle.ts

```diff
@@ -1138,6 +1156,26 @@ class Battle {
 		}
 		return false;
 	}
+	ngasActive() {
+		for (const side of this.sides) {
+			for (const active of side.active) {
+				if (active && !active.fainted && toID(active.ability) === 'neutralizinggas' && !active.volatiles['gastroacid']) {
+					return true;
+				}
+			}
+		}
+		return false;
+	}
+	anyHasAbility(...abilities: string[]) {
```

Comment:
Possibly call this `abilityActive`.

Reply:
Also if you wanted to optimize this you could, instead of using `effectiveAbility`, call `ngasActive` once and then manually check for effective abilities, skipping that loop.